docs/api.rst

```diff
@@ -0,0 +1,26 @@
+edx_api_doc_tools package
+============================
```

Comment:
Note to self: I need to clean up some of my docstrings so that they render better on ReadTheDocs.

Reply:
I've tweaked some of the docstrings.